EventSubscriber/ResponseSubscriber.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Ise\WebSecurityBundle\EventSubscriber;
+
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class ResponseSubscriber implements EventSubscriberInterface
+{
+    private $active;
+    public function __construct(ContainerInterface $container)
+    {
+        $this->active = $container->getParameter('ise_security.coop.active'); 
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            KernelEvents::RESPONSE => [
+                ['responseEvent', 0],
+            ]
+        ];
+    }
+
+    public function responseEvent(ResponseEvent $event)
+    {
+        $response = $event->getResponse();
+        if($this->active) {
```

Comment:
Is this just an initial pass? It feels as if this could be encapsulated as a policy. Maybe its own class, but maybe just a configurable set of parameters in this class.

Reply:
This is just a first pass, the intention is to have the individual header values be set based on the config, as opposed to it being a binary switch.